torchvision/prototype/features/_feature.py

```diff
@@ -16,13 +16,14 @@ def __new__(
         device: Optional[Union[torch.device, str, int]] = None,
         requires_grad: bool = False,
     ) -> F:
-        return cast(
-            F,
-            torch.Tensor._make_subclass(
-                cast(_TensorBase, cls),
-                torch.as_tensor(data, dtype=dtype, device=device),  # type: ignore[arg-type]
-                requires_grad,
-            ),
+        return (
+            torch.as_tensor(  # type: ignore[return-value]
+                data,
+                dtype=dtype,
+                device=device,  # type: ignore[arg-type]
+            )
+            .as_subclass(cls)  # type: ignore[arg-type]
+            .requires_grad_(requires_grad)
```

Comment:
Given that we no longer use `torch.Tensor._make_subclass`, we could also drop the `required_grad` here. 

On one hand we don't expect users to set `requires_grad=True` given that the features are inputs rather than parameters. Plus, even if someone needs this, a simple `.requires_grad_(True)` call is sufficient.

On the other hand, we would diverging from the other tensor creation ops. For example, the ubiquitous [`torch.tensor`](https://pytorch.org/docs/master/generated/torch.tensor.html) takes `requires_grad`. Still, it also takes the `pin_memory` parameter, which we currently also don't support.

Thoughts?

Reply:
Features can require gradients in some ML problems, for example adversarial attacks.

Reply:
Let's keep it then.